project/encefal/models.py

```diff
@@ -201,6 +205,23 @@ def __unicode__(self):
     ('REMB', 'Remboursé'),
 )
 
+### Field for prices ###
+### verify if the price is rounded 
+### (eg. 10.03 becomes 10.05
+class PriceField(models.DecimalField):
+	def validate(self, value, form):
+		"Check if the value is a valid Canadian price"
```

Comment:
In canada since we removed the 1 cent, the prices are now rounded to the nearest 5 cent multiple. By that comment, i simply mention that law as "valid canadian price".